lib/cisco_node_utils/cmd_ref/fabricpath.yaml

```diff
@@ -0,0 +1,178 @@
+# fabricpath
+---
+# Fabricpath feature is not available on N3K and N9K
+_exclude: [/N3/, /N9K/]
+
+aggregate_multicast_routes:
+  _exclude:
+    - /N5K/
+  kind: boolean
+  config_get: "show run fabricpath"
+  config_get_token: '/^fabricpath multicast aggregate\-routes$/'
+  config_set: "<state> fabricpath multicast aggregate-routes"
+  default_value: false
+  
+allocate_delay:
+  kind: int
+  config_get: "show fabricpath timers"
+  config_get_token: '/^Allocate Delay Timer\s+:\s+(\d+)/'
+  config_set: "<state> fabricpath timers allocate-delay <delay>"
+  default_value: 10
+
+auto_switch_id: 
+  kind: int
+  config_get: "show fabricpath switch-id"
+  config_get_token: '/^\*\s+(\d+).*No$/'
+
+feature:
+  config_get: "show feature-set"
+  config_get_token: '/^fabricpath[\s]+[\d]+[\s]+(\S+)/'
+  config_set: "<state> feature-set fabricpath"
+
+feature_install:
+  config_set: "<state> install feature-set fabricpath"
+
+graceful_merge:
+  auto_default: false
+  kind: boolean
+  config_get: "show run fabricpath"
+  config_get_token: '/^fabricpath graceful\-merge disable$/'
+  config_set: "<state> fabricpath graceful-merge disable"
+  default_value: true
```

Comment:
IMO this prop should either be renamed `graceful_merge_disable` or the default_value should be false...

Note that the getter returns true when it matches 'disable'... but the graceful-merge execution state is actually false in that case. 

I'm looking at this from the view of graceful-merge state, not grace-merge disable state.

```
(config)# sh run fabricpath all | i graceful-merge
(config)#
(config)# fabricpath graceful-merge disable
(config)# sh run fabricpath all | i graceful-merge
fabricpath graceful-merge disable

(config)# no fabricpath graceful-merge disable
(config)# sh run fabricpath all | i graceful-merge
(config)# 
```